dashboard/test/factories.rb

```diff
@@ -42,7 +42,7 @@
 
   factory :user do
     birthday Date.new(1991, 3, 14)
-    sequence(:email) {|n| "testuser#{n}@example.com.xx"}
+    sequence(:email) {|n| "testuser#{n}_#{SecureRandom.hex}@example.com.xx"}
```

Comment:
Fair point. Still, I prefer if the seed data were consistent and the seed task idempotent. Maybe that's too much work here and not worth it as long as we can have a consistent experience between runs with the known organizer & facilitator.